superset-frontend/plugins/plugin-chart-echarts/src/components/Echart.tsx

```diff
@@ -123,24 +134,52 @@ function Echart(
     getEchartInstance: () => chartRef.current,
   }));
 
+  const locale = useSelector(
+    (state: ExplorePageState) => state?.common?.locale ?? 'en',
```

Comment:
nit: Instead of having a magic string in the codebase, it would be nice to move this to `src/constants.ts` as `DEFAULT_LOCALE = 'en';` or similar.